contracts/operators/StakeDAO/StakeDaoCurveStrategyOperator.sol

```diff
@@ -0,0 +1,379 @@
+            tokenBalanceBefore,
+            minAmountOut
+        );
+    }
+
+    /// @notice Withdraw the LP token from the StaeDAO strategy,
+    ///         remove the liquidity from the Curve pool
+    ///         (using int128 for the curvePool.remove_liquidity_one_coin
+    ///         coin index parameter) and receive one of the
+    ///         curve pool token
+    /// @param strategy The StaeDAO strategy address to withdraw from
+    /// @param amount The amount to withdraw
+    /// @param outputToken Output token to receive
+    /// @param minAmountOut The minimum of output token expected
+    /// @return amounts Array of amounts :
+    ///         - [0] : The token received amount
+    ///         - [1] : The strategy token deposited amount
+    /// @return tokens Array of token addresses
+    ///         - [0] : The token received address
+    ///         - [1] : The strategy token deposited address
+    function withdraw128(
+        address strategy,
+        uint256 amount,
+        address outputToken,
+        uint256 minAmountOut
+    ) external payable returns (uint256[] memory amounts, address[] memory tokens) {
+        require(amount != 0, "YCVO: INVALID_AMOUNT");
+
+        (address pool, uint96 poolCoinAmount, address lpToken) = operatorStorage.strategies(strategy);
+        require(pool != address(0), "YCVO: INVALID_STRATEGY");
+
+        uint256 strategyBalanceBefore = IERC20(strategy).balanceOf(address(this));
+        uint256 tokenBalanceBefore = IERC20(outputToken).balanceOf(address(this));
+
+        _withdrawAndRemoveLiquidity128(
+            strategy,
+            amount,
+            ICurvePool(pool),
+            IERC20(lpToken),
+            poolCoinAmount,
+            outputToken
+        );
+
+        (amounts, tokens) = CurveHelpers.getOutputAmounts(
+            IERC20(strategy),
+            strategyBalanceBefore,
+            amount,
+            IERC20(outputToken),
+            tokenBalanceBefore,
+            minAmountOut
+        );
+    }
+
+    /// @notice Withdraw the LP token from the StakeDAO strategy,
+    ///         remove the liquidity from the Curve pool
+    ///         (using uint256 for the curvePool.remove_liquidity_one_coin
+    ///         coin index parameter) and receive one of the
+    ///         curve pool token
+    /// @param strategy The StakeDAO strategy address to withdraw from
+    /// @param amount The amount to withdraw
+    /// @param outputToken Output token to receive
+    /// @param minAmountOut The minimum of output token expected
+    /// @return amounts Array of amounts :
+    ///         - [0] : The token received amount
+    ///         - [1] : The strategy token deposited amount
+    /// @return tokens Array of token addresses
+    ///         - [0] : The token received address
+    ///         - [1] : The strategy token deposited address
+    function withdraw256(
+        address strategy,
+        uint256 amount,
+        IERC20 outputToken,
```

Comment:
In `withdraw128`,  the `outputToken` is an `address`    
I think we should standardize function signatures